Library/Formula/libnfc.rb

```diff
@@ -31,14 +32,27 @@ def install
 end
 
 __END__
+diff --git a/libnfc/buses/uart.c b/libnfc/buses/uart.c
```

Comment:
It was in the previous version, I just changed the patch for 1.7.1. Maybe some USB-serial adapter which the previous author has uses one of these device names.